### PR TITLE
Updating Token Hook Guide to single-page

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Converting the current Token Hook multi-page guide to a single-page guide.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-438767](https://oktainc.atlassian.net/browse/OKTA-438767)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
+---
+title: Token Inline Hook
+excerpt: Learn how to easily implement a Token Inline hook
+layout: Guides
+---
+
+<StackSelector />
+
+This guide provides a working example of an Okta Token Inline Hook. It uses the website [Glitch.com](https://glitch.com) to act as an external service and receive and respond to Token Inline Hook calls.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:9]:
```suggestion
This guide provides a working example of an Okta Token Inline Hook. It uses the website [Glitch.com](https://glitch.com) to act as an external service to receive and respond to Token Inline Hook calls.
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+---
+
+**Learning outcomes**
+
+* Understand the Okta Token Inline Hook calls and responses.
+* Implement a simple working example of a Token Inline Hook with a Glitch.com project, which acts as an external service.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:16]:
```suggestion
* Implement a simple working example of a Token Inline Hook with a Glitch.com external service.
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+* Test the Token Inline hook.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:17]:
```suggestion
* Test the Token Inline Hook.
```
"Hook" was previously capitalized, so I'm following the convention.

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+* A Node.js Express framework sample application. This guide works with the sample application in the Sample code section below.
+
+**Sample code**
+
+* [Okta Token Inline Hook Example](https://glitch.com/edit/#!/okta-inlinehook-tokenhook)
+* [Express Sample Applications for Okta](https://github.com/okta/samples-nodejs-express-4)
+
+---
+
+The Token Inline Hook can be used to customize the authorization code flow that occurs between an application and the Okta org used for authentication.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:32]:
```suggestion
The Token Inline Hook can be used to customize the Authorization Code flow that occurs between an application and the Okta org used for authentication.
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+
+4. From the **Use this inline hook** drop-down menu, select the Token Inline Hook you activated ("Patient Token Hook").
+
+5. Click **Update Rule**.
+
+The Token Inline Hook is now ready for triggering when the default policy rule is invoked from an authentication request.
+
+## Extend the sample Express application
+
+The following code extends the local sample Node.js Express application to display the results of the Token Inline Hook claim addition. This step is optional. The Token Inline Hook is functional and the results of the implementation are available in the external service logs, as well as the System Log on your Okta org. But this extension is nice to have!
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:124]:
```suggestion
The following code extends the local sample Node.js Express application to display the results of the Token Inline Hook claim addition. This step is optional. The Token Inline Hook is functional and the results of the implementation are shown in the external service logs, as well as in the system logs on your Okta org. But this extension is nice to have!
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+
+<StackSelector snippet="extend-application" noSelector/>
+
+## Test the Token Inline Hook
+
+The Token Inline Hook is now ready for testing. You now have the following applications configured:
+
+- The Okta-Hosted-Login sample application (`samples-nodejs-express-4`) is ready to authenticate users from your Okta org.
+- The external service (Glitch.com project) is ready with code to receive and respond to an Okta Token Inline Hook call.
+- The Okta org is setup to call the external service when a Token Inline Hook is triggered by a user sign-in from the Okta-Hosted-Login sample application, and ready to receive a response.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:144]:
```suggestion
- The Okta org is set up to call the external service when a Token Inline Hook is triggered by a user sign-in from the Okta-Hosted-Login sample application, and ready to receive a response.
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+
+>**Note:** Make sure you have users assigned to your application and at least one user is part of the [Patients data store](/docs/guides/token-inline-hook/#check-against-data-store/) in your Glitch application.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:146]:
```suggestion
>**Note:** Make sure you have users assigned to your application and at least one user is part of the [Patients data store](/docs/guides/token-inline-hook/#check-against-the-data-store/) in your Glitch application.
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+4. Navigate to your Glitch.com project and open the Console Logs window (**Tools** > **Logs**).
+
+5. Return to your sample application and sign in with an Okta user who is NOT in the Patients data store.
+
+    * The user should sign in as normal; only the user name should appear in the Glitch log window.
+    * If you extended the sample application, click `My Profile` in the left-hand navigation pane. Only the the user info claims are included in the table.
+
+6. Sign out of the sample application, and now sign in with an Okta user who IS in the Patients data store.
+
+    * The user should sign in as normal; however, this user should have a patient ID displayed in the Glitch console output, as well as a successful implementation record of the Token Inline Hook, available for review in your Okta org's System Log (**Reports** > **System Log**).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:165]:
```suggestion
    * The user should sign in as normal; however, this user should have a patient ID displayed in the Glitch console output, as well as a successful implementation record of the Token Inline Hook, available for review in your Okta org System Log (**Reports** > **System Log**).
```

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+
+Review the following guides to implement other Inline or Event Hook examples:
+
+* [Event Hook](/docs/guides/event-hook-implementation/)
+* [Registration Inline Hook](/docs/guides/registration-inline-hook/)
+* [Password Import Inline Hook](/docs/guides/password-import-inline-hook/)
+
+## See also
+
+For further reference data on the Token Inline Hook, see: [Token Inline Hook](/docs/reference/token-hook/).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:180]:
```suggestion

For further reference data on the Token Inline Hook, see [Token Inline Hook](/docs/reference/token-hook/).
```

**Review by vanngo-okta — APPROVED** [review]
A few nit suggestions. Everything else LGTM!  🚀

File: packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md (hunk 3 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/token-inline-hook/main/index.md:17]: Good catch. Thanks.

**Pull request merged**